### PR TITLE
Long-overdue post-acquisition updates

### PR DESCRIPTION
**Pull request by directhex**
* Try to scrub Xamarin Studio references, in favour of VSMac
* New download page, in the style of mono-project.com

### CODE REVIEW

File: download/alpha.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+layout: base
+title: Alpha Download
+navgroup: download
+redirect_from:
+  - /alpha/
+  - /Alpha/
```
Comment (akoeplinger) [inline download/alpha.md]: we don't need these redirects for new pages (they were only there for not breaking the old MediaWiki urls on mono-project.com)

File: download/alpha.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    <dl id="mono-download" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
+      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;Mac OS X</a></dd>
+      <dd><a href="#download-lin"><i class="fa fa-linux"></i>&nbsp;Linux</a></dd>
+      <dd><a href="#download-win"><i class="fa fa-windows"></i>&nbsp;Windows</a></dd>
+    </dl>
+
+    <div class="tabs-content" id="mono-download-panel">
+      <div class="panel content active" id="download-mac" style="padding: 20px 10px 10px 10px">
+        <h3>Visual Studio for Mac is available as a Mac Package (.pkg)</h3>
```
Comment (akoeplinger) [inline download/alpha.md]: VSMac is a .dmg, not a .pkg (applies to beta/stable too).

File: download/alpha.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <div>
+          <a href="{{ site.data.alpharelease.monodevelop_mac_url }}" class="button radius"><i class="fa fa-download"></i> Download Visual Studio for Mac</a>
+          <p>Supported on Mac OS X 10.7 and later.</p>
```
Comment (akoeplinger) [inline download/alpha.md]: We might want to double check with the VSMac folks on this.

File: download/alpha.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+          <p>Supported on any distribution with <a href="https://flatpak.org">Flatpak</a> 0.8.3+</p>
+          <hr />
+          <p>Currently this download only works on x86-64 operating systems. We plan to offer x86 and ARM64 builds in the future.</p>
+        </div>
+      </div>
+      <div class="panel content" id="download-win" style="padding: 20px 10px 10px 10px">
+        <h3>MonoDevelop for Windows is available from source only</h3>
+        <p>Please refer to the <a href="/developers/building-monodevelop">building guide</a> for more information about how to install and configure your MonoDevelop.</p>
+        <hr />
+        <h5>GTK# for .NET</h5>
```
Comment (akoeplinger) [inline download/alpha.md:40]: remove this or point to http://www.mono-project.com/download/#download-win so we don't need to update it in two places (applies to beta/stable too)

File: download/index.md [hunk 5]
```diff
@@ -1,91 +1,83 @@
...
+navgroup: download
 redirect_from:
+  - /downloads/
+  - /Downloads/
   - /Download/
-  - /Download/Windows_Preview/
-  - /Download/Mac_MonoTouch/
-  - /Download/Mac_Preview/
-  - /MonoDevelopWin32/
-  - /download/trunk_builds/
```
Comment (akoeplinger) [inline download/index.md]: we should keep these redirects so existing links in the wild still work.

File: download/index.md [hunk 6]
```diff
@@ -1,91 +1,83 @@
 ---
+layout: base
 title: Download
+navgroup: download
 redirect_from:
+  - /downloads/
+  - /Downloads/
```
Comment (akoeplinger) [inline download/index.md]: remove these new redirects (we didn't have these urls before so no need to introduce it)

**Review by akoeplinger — CHANGES_REQUESTED** [review]

File: download/alpha.md (hunk 3 above)
Comment (mrward) [reply download/alpha.md]: I think it is 10.11. That version is in the Info.plist and is specified on the [system requirements web page](https://www.visualstudio.com/en-us/productinfo/vs2017-system-requirements-mac)

**Review by akoeplinger — APPROVED** [review]

**Review by mrward — APPROVED** [review]

**Pull request merged**